src/pages/DashboardPage.tsx

```diff
@@ -26,6 +26,17 @@ function DashboardPage(props: Props) {
     setTitle(t('dashboard.title'));
   }, []);
 
+  var current = new Date();
```

Comment:
In plaats van `var` kun je hier het beste `const` gebruiken (van "constant"). Deze waarde mag dan niet meer veranderen.